tests/jointcalTestBase.py

```diff
@@ -0,0 +1,311 @@
+# See COPYRIGHT file at the top of the source tree.
+
+from __future__ import division, absolute_import, print_function
+
+import numpy as np
+import collections
+import os
+
+from lsst.meas.astrom import LoadAstrometryNetObjectsTask, LoadAstrometryNetObjectsConfig
+from lsst.afw import geom, table
+
+from lsst.jointcal import jointcal
+
+arcsec_per_radian = 206265.
+
+
+class JointcalTestBase(object):
```

Comment:
I was surprised to see this subclassing from `object` rather than `TestCase`, because it has very `TestCase`-like functionality (`setUp`, `tearDown`).  Is the choice to make it a mixin instead of direct subclassing because of the new test regime?  Maybe the new test finder just grabs everything that subclasses from `TestCase`?  But I don't think that would really matter here, except it would provide an additional do-nothing `TestCase`.